lexicon/lexicon-invalid.json

```diff
@@ -22,5 +22,66 @@
         }
       }
     }
+  },
+  {
+    "name": "defined unknown",
+    "lexicon": {
+      "lexicon": 1,
+      "id": "example.lexicon",
```

Comment:
Note that the TS implementation rejects those as the NSID does not contain 3 segments.